csrankings-f.csv

```diff
@@ -29,7 +29,7 @@ Fabio Gadducci,University of Pisa,http://www.di.unipi.it/~gadducci,ugYkdyYAAAAJ
 Fabio Galasso,Sapienza University of Rome,https://fgalasso.bitbucket.io,2gSuGBEAAAAJ
 Fabio Kon,USP,http://www.ime.usp.br/~kon,VE8BW84AAAAJ
 Fabio Luiz Usberti,UNICAMP,http://www.ic.unicamp.br/~fusberti,ELBC6qsAAAAJ
-Fabio Massacci,University of Trento,https://fabiomassacci.github.io,gC_ZVPgAAAAJ
+Fabio Massacci,VU Amsterdam,https://fabiomassacci.github.io,gC_ZVPgAAAAJ

```

Comment:
Web page says this faculty member has two part-time affiliations, which is not permitted. Please clarify.

Reply:
Not so sure about this. Seems he has 2 part time postions in both universities and signs both the two universities in his newest papers. Have canceled this changing.

Reply:
Please go ahead and delete his entry. Thanks.

Reply:
Deleted.